spring-petclinic/src/main/java/org/springframework/samples/petclinic/owner/OwnerController.java

```diff
@@ -20,9 +20,11 @@
 
 import javax.validation.Valid;
 
+import org.aspectj.lang.annotation.Aspect;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Component;
```

Comment:
We can remove the following imports as they are not used within the class.

import org.aspectj.lang.annotation.Aspect;
import org.springframework.stereotype.Component;